src/extension.ts

```diff
@@ -48,8 +53,8 @@ export async function activate(
   return Promise.resolve();
 }
 
-function initStartup(context: vscode.ExtensionContext) {
-  initReporter(
+async function initStartup(context: vscode.ExtensionContext) {
+  await initReporter(
```

Comment:
replace the `await` with `return`

Reply:
what? i cant return it because this is not the last line in this function...?